corehq/apps/importer/util.py

```diff
@@ -95,6 +97,25 @@ def from_request(cls, request):
 
         )
 
+
+def open_workbook(filename=None, **kwargs):
+    _soft_assert = soft_assert(notify_admins=True)
+    try:
+        return xlrd.open_workbook(filename=filename, **kwargs)
+    except xlrd.XLRDError as e:
+        message = unicode(e)
+        if message == u'Workbook is encrypted':
+            raise ImporterExcelFileEncrypted(message)
+        else:
+            _soft_assert(False, 'displaying XLRDError directly to user', e)
+            raise ImporterExcelError(message)
```

Comment:
I think another way to reraise the original exception would be simply:

```python
except xlrd.XLRDError as e:
    if ...
    else:
        raise
```
but that's not what I want to do, because that would raise an `XLRDError` which is an internal library error type. (Presumably if we supported the other excel version, we'd use a different library and want to present their errors the same way, but even if that never happens this encapsulation makes it explicit what we're raising.)

Did you maybe mean
```python
raise `ImporterExcelError`, exc_value, exc_tb
```
? I don't know if that's a convention or if it even works; never seen that before.